src/domain/fiat/primitives.ts

```diff
@@ -0,0 +1,4 @@
+export const DisplayCurrency = {
```

Comment:
This isn't the right list is it? We support a lot more display currencies so I'm confused.

Reply:
We use a strange hybrid type for this that's a union of 2 values from an "enum" and our usual unique symbol (see [here](https://github.com/GaloyMoney/galoy/blob/90294698bbc8fdc00b8780e296f11f19fb11879f/src/domain/fiat/index.types.d.ts#L6)). It's also just a scalar in the graphql layer vs. `WalletCurrency` which is an enum there.

We're loading these currency types in dynamically from config, so I think we can probably decide now if we'd like to just get rid of the enum part of the type.

Reply:
Don't see the point in a hybrid. Would load it all dynamically

Reply:
Fixed in #3193